tools/bin/hawqregister

```diff
@@ -131,6 +130,18 @@ def option_parser_yml(yml_file):
 
 def create_table(dburl, tablename, schema_info, fmt, distrbution_policy, file_locations):
     try:
+        query = "select count(*) from pg_class where relname = '%s'" % tablename.split('.')[-1]
```

Comment:
Maybe we need to consider the upper/lower cases of tablename.  All the table name stored in HAWQ are converted to lower case. So I think we need to convert tablename to lower case and then do comparison.